base/ca/src/main/java/com/netscape/cms/servlet/request/ProcessCertReq.java

```diff
@@ -1487,10 +1487,7 @@ protected int grant_privileges(
         boolean grantDRMAgent =
                 httpParams.getValueAsBoolean(GRANT_DRMAGENT_PRIVILEGE, false);
 
-        if (!grantTrustedMgr &&
-                !grantCMAgent && !grantRMAgent && !grantDRMAgent) {
-            return 0;
-        } else {
+        if (grantTrustedMgr || grantCMAgent || grantRMAgent || grantDRMAgent) {
```

Comment:
Should we keep the if-block since it returns and unindent the else-block instead?